conmon-rs/server/src/child_reaper.rs

```diff
@@ -149,6 +152,34 @@ impl ChildReaper {
         Ok((grandchild_pid, token))
     }
 
+    fn check_child_processes() {
+        debug!("Checking child processes");
+        let pid = Pid::from_raw(-1);
+        loop {
+            match waitpid(pid, WaitPidFlag::WNOHANG.into()) {
+                Ok(WaitStatus::Exited(p, code)) => {
+                    debug!("PID {p} exited with status: {code}");
+                    break;
+                }
+                Ok(WaitStatus::StillAlive) => {
```

Comment:
Do we also want to match here on  `WaitStatus::Stopped()` and `WaitStatus::Continued()`? I am thinking about a case when the process suddenly _goes away_ for reasons outside of conmon-rs' influence.

Reply:
On the other hand, perhaps leaving it `Ok()` is fine, to skip over, so to speak.

Reply:
Basically, I am not sure if `WaitStatus::StillAlive` covers stopped and continued process, and if not, then whether we should break here for such or continue waiting? @saschagrunert, thoughts?

Reply:
Makes sense. Thank you!